.github/workflows/test.yml

```diff
@@ -1,17 +1,67 @@
-name: "build-test"
-on: # rebuild any PRs and main branch changes
+on:
```

Comment:
whenever you're fixing tests raise for master first.

Reply:
Okay. will take care next time.